CHANGES_NEXT_RELEASE

```diff
@@ -1,6 +0,0 @@
-- Fix: update tests to not create initial entity when append mode is enabled
-- Remove: expressionLanguage field (as it it not longer needed, due to only one expression language is supported, after the removal of legacy expressions in iota-node-lib 3.2.0) (iota-node-lib#1384)
-- Fix: update tests according new default value for appendMode
-- Fix: use 'options=upsert' when update ngsiv2 CB entities and appendMode is enabled (iotagent-node-lib#956)
-- Fix: do not propagate group config (timestamp and explicitAttrs) to autoprovisioned devices (at database level) (iotagent-node-lib#1377)
-- Remove: RPM stuff (no longer used)
```

Comment:
Should be moved to rpm/SPECS/iotaul.spec

Reply:
rpm/SPECS/iotaul.spec was removed

Reply:
Ah ok... the right file is Changelog, in the root of the repository

Reply:
fixed in https://github.com/telefonicaid/iotagent-ul/pull/622/commits/8a7e5870dd3e5e3b470c774c661b5e7db89847ba